packages/deployment/lib/lambda.js

```diff
@@ -62,12 +62,16 @@ class UpdatedLambda extends Lambda {
     }
 
     console.log(`${msg} for ${lambda.name}`);
-    return utils.zip(lambda.local, fileList)
-      .then(() => lambda)
-      .catch((e) => {
-        console.log(`Error zipping ${e}`);
-        throw (e);
-      });
+
+    try {
```

Comment:
Since this is an async function, we can use a try/catch to make it a little more explicit what is going on.